### PR TITLE
Add team members/ fixes

### PR DESCRIPTION
**Pull request by Moejoe90**

### CODE REVIEW

File: page/blocks/person.ejs [hunk 1]
```diff
@@ -9,5 +9,10 @@
     <p class="bio hidden">
         <%= bio %>
     </p>
-    <span class="pre-title hidden"><%if (typeof preTitle != 'undefined') {%>  <%= preTitle %>  <%}%></span>
```
Comment (MostafaBalata) [inline page/blocks/person.ejs]: Why did you remove the "pre title section"

Comment (Moejoe90) [inline page/blocks/person.ejs]: Why do you want to use pre-title? 
What is the context?

File: page/partials/commit_funds.ejs [hunk 2]
```diff
@@ -13,12 +13,12 @@
...
                 <div class="content">
-                    <h3><%- sections[0].title -%></h3>
-                    <p><%- sections[0].paragraph1 %></p>
-                    <p><%- sections[0].paragraph2 %></p>
+                    <h3><%- sections[1].title -%></h3>
+                    <p><%- sections[1].paragraph1 %></p>
+                    <p><%- sections[1].paragraph2 %></p>
                 </div>
             </div> <!-- end col-md-6 -->
         </div>
```
Comment (MostafaBalata) [inline page/partials/commit_funds.ejs]: why you didn't make loop over the div instead of make two identical sections?

File: page/partials/commit_funds.ejs [hunk 3]
```diff
@@ -13,12 +13,12 @@
...
-                    <%- sections[0].point -%>
+                    <%- sections[1].point -%>
                 </div>
                 <div class="content">
-                    <h3><%- sections[0].title -%></h3>
-                    <p><%- sections[0].paragraph1 %></p>
-                    <p><%- sections[0].paragraph2 %></p>
+                    <h3><%- sections[1].title -%></h3>
+                    <p><%- sections[1].paragraph1 %></p>
+                    <p><%- sections[1].paragraph2 %></p>
```
Comment (MostafaBalata) [inline page/partials/commit_funds.ejs]: why you didn't make loop over the div instead of make two identical sections?

Comment (MostafaBalata) [inline page/partials/commit_funds.ejs]: Something like this
```
<%- for( const i = 0 ; i < sections.length; i++) { %>
            <div class="col-md-5">
                <div class="number">
                    <%- sections[i].point -%>
                </div>
                <div class="content seperator">
                    <h3><%- sections[i].title -%></h3>
                        <p><%- sections[i].paragraph1 -%></p>
                        <p><%- sections[i].paragraph2 %></p>
                </div>
            </div> <!-- end col-md-6 -->
<% } %>
```

File: page/ts/index.ts [hunk 4]
```diff
@@ -70,11 +70,14 @@ $(document).ready(function () {
...
+        const domain: string = $(this).find('a.domain').text();
+        const email: string = $(this).find('p.link').text();;
+
 
-        vex.dialog.open(getPersonModal(name, image, preTitle, title, bio))
+        vex.dialog.open(getPersonModal(name, image, preTitle, title, bio, domain, email))
     });
 
-    $('body').on('click', '.close-modal', function (e) {
+    $('body').on('click', '.close-modal' , function(e){
```
Comment (MostafaBalata) [inline page/ts/index.ts]: `'.close-modal' ,` remove the space before the comma.

File: page/partials/commit_funds.ejs (hunk 3 above)
Comment (Moejoe90) [inline page/partials/commit_funds.ejs]: I don't think that's a good idea take notice that styles might change any minute to something completely different, they are identical now but its not guaranteed. Doing something like this might create unneeded over head

File: page/blocks/person.ejs (hunk 1 above)
Comment (MostafaBalata) [inline page/blocks/person.ejs]: This is from the design.
screen shot 2017-08-23 at 3 57 51 pm

check the word "Partner"

File: page/ts/index.ts (hunk 4 above)
Comment (Moejoe90) [inline page/ts/index.ts]: will do

File: page/blocks/person.ejs (hunk 1 above)
Comment (Moejoe90) [inline page/blocks/person.ejs]: Will bring it back

File: content.yml [hunk 5]
```diff
@@ -60,15 +60,165 @@ team:
...
-      "Lorem ipsum dolor sit amet, consectetur adipiscing elit,
-      sed do eiusmod tempor incididunt ut labore et dolore magna aliqua."
+   - name: "Zoe Adamovicz"
+     title: "CEO and Co-Founder"
+     image: "../assets/ZoeAdamovicz_bw.jpg"
+     preTitle: ""
+     domain: ""
+     email: "writeme@neufund.org"
+     bio:  |
+       "Zoe Adamovicz is the CEO and co-founder at Neufund. She is an experienced entrepreneur and occasional angel investor.
```
Comment (krzkaczor) [inline content.yml]: looks good. Are new lines preserved in final output?

Comment (Moejoe90) [inline content.yml]: Yes, they are something like this
image
